src/sass/components/_collaborators.scss

```diff
@@ -0,0 +1,43 @@
+// Collaborators
+// =============
+
+// Add/Edit Collaborator Modal
+// ---------------------------
+.collaborator-username {
+  margin-left: $spacing-x-small;
+
+  .collaborator-button.is-selected & {
+    color: $color-text-default;
+  }
+}
+
+.collaborator-button {
+  &.is-selected {
+    background-color: $color-background-row-active;
+  }
+
+  &:not(.is-selected) {
+    &:hover {
+      border: 1px solid;
+    }
+  }
+
+  // don't show a border on tables with checkboxes
+  .table-row-targets & {
+    &:hover {
+      border-color: transparent;
+    }
+
+    &:focus {
```

Comment:
These are currently keyboard-focus targets in the tab-order,  with no focus style

Reply:
Yeah, I think @stacyk was aware of that and did it intentionally?

> I don't like the double tabbing, but I know I asked for the name to be a clickable label, and because this is in a table we can't wrap the label around the checkbox and text.

Reply:
Seems like if we're going to leave the double-tab there should be *some* focus style that goes with it?

Reply:
Added focus styles back.

![Screenshot 2020-01-31 21 59 02](https://user-images.githubusercontent.com/552316/73585956-e9586180-4474-11ea-83f1-d757ffbdabfe.png)